Library/Formula/ibex.rb

```diff
@@ -0,0 +1,35 @@
+class Ibex < Formula
+  desc "C++ library for constraint processing over real numbers."
+  homepage "http://www.ibex-lib.org/"
+  url "http://www.ibex-lib.org/sites/default/files/ibex-2.1.16.tar.gz"
+  sha256 "d92ff32f14d27ad7b390ae693beb311b58cf6babccac85bbdaa5f5d0b8648845"
+
+  option "with-java", "Build Java bindings for Choco solver."
+
+  depends_on :java => ["1.8+", :optional]
+  depends_on "bison" => :build
+  depends_on "flex" => :build
+  depends_on "pkg-config" => :build
+
+  def install
+    if build.with? "java"
```

Comment:
Rather than duping the whole block here let's go with something like:

```ruby
    args = ["--prefix=#{prefix}"]

    if build.with? "java"
      args << "--with-jni"
    else
      args << "--enable-shared"
    end

    system "./waf", "configure", *args
```